core/src/epicli/data/common/defaults/configuration/postgresql.yml

```diff
@@ -1,62 +1,65 @@
 kind: configuration/postgresql
-title: PostgreSQL
 name: default
 specification:
-  replication:
-    enabled: no
   config_file:
     parameter_groups:
-      - name: CONNECTIONS AND AUTHENTICATION
-        subgroups:
-          - name: Connection Settings
-            parameters:
-              - name: listen_addresses
-                value: "'*'"
-                comment: listen on all addresses
-          - name: Security and Authentication
-            parameters:
-              - name: ssl
-                value: 'off'
-                comment: to have the default value also on Ubuntu
-      - name: RESOURCE USAGE (except WAL)
-        subgroups:
-          - name: Kernel Resource Usage
-            parameters:
-              - name: shared_preload_libraries
-                value: SET_BY_AUTOMATION
-                comment: set by automation
-      - name: ERROR REPORTING AND LOGGING
-        subgroups:
-          - name: Where to Log
-            parameters:
-              - name: log_directory
-                value: "'/var/log/postgresql'"
-                comment: to have standard location for Filebeat and logrotate
-              - name: log_filename
-                value: "'postgresql.log'"
-                comment: to use logrotate with common configuration
-  additional_components:
-    pgbouncer:
-      enabled: no
+    - name: CONNECTIONS AND AUTHENTICATION
+      subgroups:
+      - name: Connection Settings
+        parameters:
+        - comment: listen on all addresses
```

Comment:
Any reason to change the order? Originally names were at the top, which IMHO is more readable, than starting from comments.

Also indentations have been changed which is not good for comparing revisions.

And quotation as well, for example originally we had:
`"'/var/log/postgresql'"`, now we have `'''/var/log/postgresql'''`
IMHO old format was better than **triple** apostrophes.